plugins/s3-viewer-backend/package.json

```diff
@@ -15,10 +15,17 @@
   "repository": {
     "type": "git",
     "url": "https://github.com/spreadshirt/backstage-plugin-s3",
-    "directory": "plugins/backstage-plugin-s3-viewer-backend"
+    "directory": "plugins/s3-viewer-backend"
   },
   "backstage": {
-    "role": "backend-plugin"
+    "role": "backend-plugin",
+    "pluginId": "s3-viewer",
+    "pluginPackages": [
+      "@spreadshirt/backstage-plugin-s3-viewer",
+      "@spreadshirt/backstage-plugin-s3-viewer-backend",
+      "@spreadshirt/backstage-plugin-s3-viewer-common",
+      "@spreadshirt/backstage-plugin-s3-viewer-node"
```

Comment:
Do we need to include all packages for every sub-package?  Seems a bit counter-intuitive, I'd expect to only see `@spreadshirt/backstage-plugin-s3-viewer-backend` here.

Reply:
Looks like it. This is something automated. Which is fixed by running `yarn backstage-cli repo fix --publish`

Reply:
How odd, but if the tool says so...

Reply:
I also thought it when I did this for the vault plugin. Maybe there's some refinement being needed and at some point it's required to update this again